### PR TITLE
add desktop version

### PR DESCRIPTION
**Pull request by devMahmoud**
desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,21 +19,29 @@
...
+        <div class="intro">
+            <h1 class="name">Hey There.<br>I’m Mahmoud</h1>
+            <h2 class="job-title">I’am a Software Developer</h2>
+            <p>I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
```
Comment (smunozmo) [inline index.html:44]: - [x] Good job setting up your desktop version! However, your design needs some adjustments in order to look similar to the Figma template:

Captura de pantalla 2022-03-25 a la(s) 7 42 02 p  m

Try adding some space between the text and the social icons and making your paragraph narrower.

File: index.html [hunk 2]
```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
             </div>
             
             <div class="first-card">
```
Comment (smunozmo) [inline index.html:85]: - [x] Your works section is almost ready, let's try to make it even better, like the Figma template:

Captura de pantalla 2022-03-25 a la(s) 7 46 37 p  m

- Please add some space around the title and line.
- Kindly reduce the space between the image and the text info.

File: index.html [hunk 3]
```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
...
-                <img class="first-project" src="img/top-proj.png" alt="project-screenshot">
+                <div class="first-project"></div>
 
                 <div class="first-card-content">
```
Comment (smunozmo) [inline index.html:88]: - [x] Cards are almost there, I would suggest reducing the space between them to make your design more similar to the Figma design:

Captura de pantalla 2022-03-25 a la(s) 7 51 29 p  m

File: index.html [hunk 4]
```diff
@@ -212,7 +220,7 @@ <h2 class="about-title">About me</h2>
         </div>
 
         <div class="devider"></div>
-
+
```
Comment (smunozmo) [inline index.html:223]: - [x]  About section is getting there, but we need to make some changes to match the Figma design:

Captura de pantalla 2022-03-25 a la(s) 7 59 01 p  m

- Please add some space on the left side of the About title and its paragraph.
- The about description should be split into several lines and narrower.
- Try centering the divider line and give it some space around.
- `Language, Framework and Skills` titles should be aligned horizontally.

File: index.html [hunk 5]
```diff
@@ -212,7 +220,7 @@ <h2 class="about-title">About me</h2>
         </div>
 
         <div class="devider"></div>
-
+        
         <div class="skills">
             <ul class="lang-list">
                 <li><ul class="language list-item1"><li>Language</li></ul></li>
```
Comment (smunozmo) [inline index.html:226]: - [x] Let's make the contact form look more like the Figma template:

Captura de pantalla 2022-03-25 a la(s) 8 02 10 p  m

- Let's start by adding some space on the left side of the text.
- Please align the text to the left, it is currently using a center alignment.
- Kindly remove the big blank space between the text and the form.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @devMahmoud, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
             </div>
             
             <div class="first-card">
-                <img class="first-project" src="img/top-proj.png" alt="project-screenshot">
+                <div class="first-project"></div>
```
Comment (mmsesay) [inline index.html:86]: - [x] I think this issue is still not fixed yet. See the screenshot of your project and please try to increase the width of the cards for the desktop version.

Your Project design|
:------------------
a1
-------:|

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @devMahmoud,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (devMahmoud) [reply index.html:86]: @mmsesay I fixed it. kindly check

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @devMahmoud👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**